assets/js/main.js

```diff
@@ -3,21 +3,36 @@ $("body").on({
     var submenu = $(this).find(".sub-menu");
     if (submenu && submenu[0].children.length) {
       $('.header-main').addClass('header-open');
-    }  
+    }
   },
   mouseleave: function() {
     $('.header-main').removeClass('header-open');
   }
 }, ".menu-main .submenu-item");
 
-$("body").on('click', '#user-menu--open', function(){
+$("body").on('click', '#user-menu--open', function() {
   $('.user-menu').toggleClass('open');
 });
 
-$("body").on('click', '#open-menu', function(){
+$("body").on('click', '#open-menu', function() {
   $('.header-main').addClass('open');
 });
 
-$("body").on('click', '#close-menu', function(){
+$("body").on('click', '#close-menu', function() {
   $('.header-main').removeClass('open');
-});
\ No newline at end of file
+});
+
+// modals
+
+$("#btn-modal-large").click(function() {
+  $("#modal-large").fadeIn(500);
```

Comment:
Maybe add this default delay into a var `500` What do you think?

Reply:
great idea!